cytoscape-panzoom.js

```diff
@@ -311,12 +316,26 @@ SOFTWARE.
             windowBind("mousemove", handler);
           });
 
+          $pHandle.bind("touchstart", function(e){
+            handler(e);
+
+            windowBind("touchmove", handler);
+          });
+
           $pHandle.bind("mouseup", function(){
-            donePanning();
+            donePanning("mousemove");
```

Comment:
Imho, `donePanning()` should sort this out. Easier to change/update if you just call something like `donePanning(e);`, and then let `donePanning()` determine what to unbind dynamically using e.type.

Reply:
That's kind of tough considering the event given would be a mouseup, and when we're done panning, we want to unbind the mousemove event.

Reply:
Actually nvm, I misread what you typed.